packages/allure-js-commons/src/AllureCommandStep.ts

```diff
@@ -228,7 +250,7 @@ export class AllureCommandStepExecutable implements AllureCommandStep {
             start: startDate,
             stop: new Date().getTime(),
             stage: Stage.FINISHED,
-            status: Status.PASSED,
+            status: actualStatus,
```

Comment:
what about soft assertions? did hermione or cucumber have them? how it will work?

Reply:
What the soft assertions?

Reply:
Honestly I don't know about this feature in the frameworks. Anyway, if test will be marked as failed by any reason, result will be failed too.